apps/example/package.json

```diff
@@ -8,15 +8,17 @@
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "start": "react-native start",
-    "pod:install": "pod install --project-directory=ios",
+    "pod:install:ios": "pod install --project-directory=ios",
+    "pod:install:macos": "pod install --project-directory=macos",
     "build:android": "cd android && ./gradlew assembleDebug --warning-mode all",
     "build:ios": "yarn run mkdist && react-native bundle --entry-file index.js --platform ios --dev true --bundle-output dist/main.ios.jsbundle --assets-dest dist && react-native build-ios --scheme Example --mode Debug --extra-params \"-sdk iphonesimulator CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ GCC_OPTIMIZATION_LEVEL=0 GCC_PRECOMPILE_PREFIX_HEADER=YES ASSETCATALOG_COMPILER_OPTIMIZATION=time DEBUG_INFORMATION_FORMAT=dwarf COMPILER_INDEX_STORE_ENABLE=NO\"",
+    "build:macos": "yarn run mkdist && react-native bundle --entry-file index.js --platform macos --dev true --bundle-output dist/main.macos.jsbundle --assets-dest dist",
```

Comment:
kudos :))

Reply:
that doesn't seem to build the app, could we put the xcode build command here?

Reply:
Yeah, I think I need to actually go and make sure that shows up in the React Native community CLI :D. I'll follow up on that, and meanwhile split out some more changes from this PR.

Reply:
I checked with @tido64. We need to go and implement `build-macos`. We have `run-macos`... so it shouldn't be too hard. But it might be a bit.